src/docker-images.js

```diff
@@ -1,19 +1,18 @@
 // These have to exist, so we don't bother checking if they exist on the system first
 exports.globalImages = {
-	'nginx-proxy': '10up/nginx-proxy:latest',
-	mysql: 'mysql:5',
+	'nginx-proxy': 'jwilder/nginx-proxy:alpine',
+	mysql: 'mariadb:10.3',
 	mailcatcher: 'schickling/mailcatcher',
-	phpmyadmin: 'phpmyadmin/phpmyadmin',
```

Comment:
Same here. Let's update it to use the [official image](https://hub.docker.com/_/phpmyadmin) for phpmyadmin.